example/src/main/java/io/netty/example/http2/tiles/Html.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.example.http2.tiles;
+
+import io.netty.example.http2.tiles.http.HttpServer;
+import io.netty.example.http2.tiles.http2.Http2Server;
+
+import java.util.Random;
+
+/**
+ * Static and dynamically generated HTML for the example.
+ */
+public final class Html {
+
+    public static final String IP = System.getProperty("ip", "127.0.0.1");
+
+    public static final byte[] FOOTER = "</body></html>".getBytes();
+
+    public static final byte[] HEADER = ("<!DOCTYPE html><html><head lang=\"en\"><title>Netty HTTP/2 Example</title>"
+            + "<style>body {background:#DDD;} div#netty { line-height:0;}</style>"
+            + "<link rel=\"shortcut icon\" href=\"about:blank\">"
+            + "<meta charset=\"UTF-8\"></head><body>A grid of 200 tiled images is shown below. Compare:"
+            + "<p>[<a href='https://" + url(Http2Server.PORT) + "?latency=0'>HTTP/2, 0 latency</a>] [<a href='http://"
+            + url(HttpServer.PORT) + "?latency=0'>HTTP/1, 0 latency</a>]<br/>" + "[<a href='https://"
+            + url(Http2Server.PORT) + "?latency=30'>HTTP/2, 30ms latency</a>] [<a href='http://" + url(HttpServer.PORT)
+            + "?latency=30'>HTTP/1, 30ms latency</a>]<br/>" + "[<a href='https://" + url(Http2Server.PORT)
+            + "?latency=200'>HTTP/2, 200ms latency</a>] [<a href='http://" + url(HttpServer.PORT)
+            + "?latency=200'>HTTP/1, 200ms latency</a>]<br/>" + "[<a href='https://" + url(Http2Server.PORT)
+            + "?latency=1000'>HTTP/2, 1s latency</a>] [<a href='http://" + url(HttpServer.PORT)
+            + "?latency=1000'>HTTP/1, " + "1s latency</a>]<br/>").getBytes();
+
+    private Html() {
+    }
+
+    private static String url(int port) {
+        return IP + ":" + port + "/http2";
+    }
+
+    public static byte[] body(int latency) {
+        StringBuilder sb = new StringBuilder("<div id=\"netty\">");
```

Comment:
Not a big deal...looks like the random int is generated up front and the same one is used throughout the loop building the string. You should consider translating to string up front and then you can just use `myIntAsString.length() * numerTimesToInsert`.